app/models/page.rb

```diff
@@ -135,6 +135,17 @@ def image_to_display
     primary_image || images.first
   end
 
+  def plugin_thermometers
+    plugins.collect do |x|
+      x if ['Plugins::ActionsThermometer', 'Plugins::DonationsThermometer'].include?(x.class.to_s)
```

Comment:
Better to use `kind_of?` or `is_a?`

Reply:
@osahyoun 
does using `member?` is fine?
  [Plugins::ActionsThermometer, Plugins::DonationsThermometer].member?(x.class)

if we use kind_of? and Is_a? it ends 
Plugins::ActionsThermometer.is_a?(x.class) ||  Plugins::DonationsThermometer.is_a?(x.class)
Shall I use the member? or using the || operator?

Reply:
👍